org-static-blog.el

```diff
@@ -488,16 +497,25 @@ archive headline."
      (cdr tag)
      (concat "<h1 class=\"title\">Posts tagged \"" (car tag) "\":</h1>"))))
 
+(defun org-static-blog-assemble-tags-archive-tag (tag)
+  "Assemble single TAG for all filenames."
+  (let ((post-filenames (cdr tag)))
+    (setq post-filenames
+	  (sort post-filenames (lambda (x y) (time-less-p (org-static-blog-get-date x)
```

Comment:
This switches the order of `x` and `y` relative to the previous code. Is this intentional?

Reply:
This was originally farther down, located here: https://github.com/bastibe/org-static-blog/blob/master/org-static-blog.el#L520-L521